docs/writing-a-plugin/README.md

```diff
@@ -22,29 +22,26 @@ Transform streams are streams that are readable and writable; they manipulate ob
 
 All gulp plugins essentially boil down to this:
 ```js
-var Transform = require('transform');
-
-module.exports = function() {
-  // Monkey patch Transform or create your own subclass,
-  // implementing `_transform()` and optionally `_flush()`
-  var transformStream = new Transform({objectMode: true});
-  /**
-   * @param {Buffer|string} file
-   * @param {string=} encoding - ignored if file contains a Buffer
-   * @param {function(Error, object)} callback - Call this function (optionally with an
-   *          error argument and data) when you are done processing the supplied chunk.
-   */
-  transformStream._transform = function(file, encoding, callback) {
-    var error = null,
-        output = doSomethingWithTheFile(file);
-    callback(error, output);
-  };
-
-  return transformStream;
-};
+var Transform = require('stream').Transform;
+
+-module.exports = function() {
+-  // Monkey patch Transform or create your own subclass,
+-  // implementing `_transform()` and optionally `_flush()`
```

Comment:
This shouldn't have changed.  Looks like you copy-pasted poorly and included `-`s and missed the end of the function.